src/components/EditPartnership/EditPartnership.tsx

```diff
@@ -10,11 +24,89 @@ interface EditPartnershipProps {
 
 export const EditPartnership: FC<EditPartnershipProps> = ({
   state,
-  // partnership,
+  partnership,
 }) => {
+  const [updatePartnership] = useUpdatePartnershipMutation();
+  const [deletePartnership] = useDeletePartnershipMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit = async (input: UpdatePartnershipInput) => {
+    const inputWithId: UpdatePartnershipInput = {
+      partnership: { ...input.partnership, id: partnership.id },
+    };
+    await updatePartnership({
+      variables: { input: inputWithId },
+      refetchQueries: ['ProjectPartnerships'],
+    });
+    enqueueSnackbar(
+      `Updated Partnership with ${partnership.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
```

Comment:
I'm not sure snackbars are that useful here. We have them on global create dialogs since those can be invoked anywhere and you don't immediately see the results on screen. With both edit and delete here the changes should be reflected in the list and immediately visible to the user.
What do you think?

Reply:
agree, removed